### PR TITLE
[receiver/windowsperfcounters] Returning partial errors for failures

### PR DESCRIPTION
**Pull request by chrislbs**
Returning partial errors for failure during scraping to prevent throwing out all succesfully retrieved metrics.

**Description:** Fixing a bug where a single instance of a windows performance object prevents metrics for all other valid instances from being published.

**Link to tracking Issue:** #16712

**Testing:** I added additional tests that verifies if some scrapers fail to retrieve metrics, the ones that don't fail continue to publish their metrics.

**Documentation:**

### CODE REVIEW

**Comment by fatsheep9146** [comment]
@mx-psi can you approve this running workflow?

**Comment by fatsheep9146** [comment]
The changelog should be added. @chrislbs

**Comment by chrislbs** [comment]
I'll get the changelog entry done today.

**Comment by fatsheep9146** [comment]
@chrislbs any progress?

File: receiver/windowsperfcountersreceiver/windowsperfcounters_scraper.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package windowsperfcountersreceiver // import "github.com/open-telemetry/opentel
 
 import (
 	"context"
+	"go.opentelemetry.io/collector/receiver/scrapererror"
```
Comment (dmitryax) [inline receiver/windowsperfcountersreceiver/windowsperfcounters_scraper.go:22]: Please move this import to the second group. This should solve the linter errors. Same in the test file

**Comment by dmitryax** [comment]
And please rebase

**Comment by fatsheep9146** [comment]
ping @chrislbs

**Comment by bartecargo** [comment]
This is sorely needed for Windows users.

**Pull request closed without merging**